packages/adapters-library/src/adapters/aave-v2/common/aaveBasePoolAdapter.ts

```diff
@@ -192,22 +193,20 @@ export abstract class AaveBasePoolAdapter implements IProtocolAdapter {
           tokenAddress,
         )
 
-      const protocolTokenPromise = getTokenMetadata(
+      const protocolTokenPromise = this.helpers.getTokenMetadata(
         this.getReserveTokenAddress(reserveTokenAddresses),
-        this.chainId,
-        this.provider,
-      )
-      const underlyingTokenPromise = getTokenMetadata(
-        tokenAddress,
-        this.chainId,
-        this.provider,
       )
+      const underlyingTokenPromise = this.helpers.getTokenMetadata(tokenAddress)
 
       const [protocolToken, underlyingToken] = await Promise.all([
         protocolTokenPromise,
         underlyingTokenPromise,
       ])
 
+      if (protocolToken.address === ZERO_ADDRESS) {
+        return
+      }
+
```

Comment:
Zero address was bing added as a pool address

context: seems aave removed stable interest debt token for eth